runtime-testsuite/src/org/antlr/v4/testgen/TargetConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package org.antlr.v4.testgen;
```

Comment:
:exclamation: The runtime test suite should not have knowledge of specific targets.